web/react/components/post_list.jsx

```diff
@@ -37,15 +37,25 @@ module.exports = React.createClass({
     componentDidMount: function() {
         var user = UserStore.getCurrentUser();
         if (user.props && user.props.theme) {
-            utils.changeCss('a.theme', 'color:'+user.props.theme+'; fill:'+user.props.theme+'!important;');
             utils.changeCss('div.theme', 'background-color:'+user.props.theme+';');
             utils.changeCss('.btn.btn-primary', 'background: ' + user.props.theme+';');
-            utils.changeCss('.btn.btn-primary:hover, .btn.btn-primary:active, .btn.btn-primary:focus', 'background: ' + utils.changeColor(user.props.theme, -10)  +';');
             utils.changeCss('.modal .modal-header', 'background: ' + user.props.theme+';');
             utils.changeCss('.mention', 'background: ' + user.props.theme+';');
             utils.changeCss('.mention-link', 'color: ' + user.props.theme+';');
             utils.changeCss('@media(max-width: 768px){.search-bar__container', 'background: ' + user.props.theme+';}');
         }
+        if (user.props.theme != "#000000" && user.props.theme != "#585858"){
```

Comment:
Since this isn't for 0.6.0, can you update this part to match the new style guide? You don't need to change the rest of the file, but for the new stuff you're adding, can you fix it to match that? The main things I'm seeing here are that it's using double quotes instead of single quotes and that the whitespace around brackets, curly brackets, and plus signs are off.